docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,365 @@
+      "text/plain": [
+       "<Figure size 1625.27x451.5 with 1 Axes>"
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "num_qubits = 5\n",
+    "num_time_steps = 3\n",
+    "rx_angle = 0.1\n",
+    "rzz_angle = 0.1\n",
+    "\n",
+    "ising_circuit = QuantumCircuit(num_qubits)\n",
+    "for i in range(num_time_steps):\n",
+    "    # rx layer\n",
+    "    for q in range(num_qubits):\n",
+    "        ising_circuit.rx(rx_angle, q)\n",
+    "    for q in range(1, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    # 2nd rzz layer\n",
+    "    for q in range(0, num_qubits-1, 2):\n",
+    "        ising_circuit.rzz(rzz_angle, q, q+1)\n",
+    "    ising_circuit.barrier()\n",
+    "ising_circuit.draw('mpl')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3aae282-1656-4311-9d05-87e1f80491de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Specify a backend which has fractional gates enabled, allowing a transpiler pass manager to take that into its own context\n",
+    "service = QiskitRuntimeService()\n",
+    "backend_fractional = service.least_busy(use_fractional_gates=True)\n",
+    "backend_conventional = service.least_busy(use_fractional_gates=False)\n",
+    "\n",
+    "pm_fractional = generate_preset_pass_manager(optimization_level=3, backend=backend_fractional, scheduling_method=\"alap\")\n",
+    "pm_conventional = generate_preset_pass_manager(optimization_level=3, backend=backend_conventional, scheduling_method=\"alap\")\n",
+    "\n",
+    "ising_circuit_fractional = pm_fractional.run(ising_circuit)\n",
+    "ising_circuit_conventional = pm_conventional.run(ising_circuit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d8a8315b-3077-47b6-a377-6b561a5b1fba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Draw timeline of circuit with conventional gates\n",
+    "draw_timeline(ising_circuit_conventional, idle_wires=False, time_range=(0, 500), style=IQXSimple())"
```

Comment:
are these draw_timeline functions supposed to print out an image? I don't see anything in the preview

Reply:
They will, the code won't work until the `use_fractional_gates` flag is enabled. That's part of why I put in the temporary code to show what the images will look like.